source/main/ForwardDeclarations.h

```diff
@@ -182,7 +176,6 @@ class ProceduralManager;
 class RailSegment;
 class RailGroup;
 class Replay;
-class RigsOfRods;

```

Comment:
I can't believe this was still around 😆

Reply:
Should we leave it in for some extra style points?

Reply:
Nah, it's still just deadcode.